docs/how-to/create-chiselled-docker-image.md

```diff
@@ -0,0 +1,181 @@
+# Create a chiselled Docker image
+
+Since Chisel can produce a minimal root file system, using it in Docker images
+has lots of value. This guide shows how to create a chiselled Docker image.
+
+A _chiselled_ image is one which only contains files installed by the slices and
```

Comment:
I'd remove this. it's unnecessarily conveying double meaning to the word "chiseled" vs the fact that is must be baseless (which is not true - but rather more like an intuitive pattern to follow)

Reply:
Removed.